### ISSUE TITLE
Creating single track Transition gives "More media overlap needed..."

### ISSUE THREAD

**Issue by jliljebl** [jliljebl/flowblade-forum#32]

**Comment by yichuang-zh** [comment]
Thanks for your work~

And, I feel that the operation process is a bit complicated. Will you consider adjusting it later? 
Or, there is no need to make any changes?

**Comment by jliljebl** [comment]
@yichuang-zh Yes, there is this Issue so there will improvement on making this simpler in the future: 
https://github.com/jliljebl/flowblade/issues/763

**Comment by yichuang-zh** [comment]
Great~
I am looking forward.

**Comment by pghpete** [comment]
Unfortunately this doesn't appear to work in all situations. (fedora 31, flowblade 2.4.0)

**Comment by allelopath** [comment]
Any date of a fix for this?

**Comment by jliljebl** [comment]
2.6 will have option to automatically shorten clip if possible, 2.6  should come this month.

**Comment by allelopath** [comment]
Maybe I'll understand it when I see it but ... I don't want to shorten either clip involved, I just want a transition between the two.

**Comment by jliljebl** [comment]
Please read this whole thread. I don't think I'm able offer any better explanation, hope you get the issue here.

**Comment by Cyntique** [comment]
You saved my life!! Thank you!
